packages/peregrine/lib/talons/CartPage/PriceAdjustments/useCouponCode.js

```diff
@@ -87,8 +87,24 @@ export const useCouponCode = props => {
         setIsCartUpdating
     ]);
 
+    const applyErrorMessage = useMemo(() => {
+        if (!applyError) return null;
+
+        if (applyError.graphQLErrors) {
+            // Apollo prepends "GraphQL Error:" onto the message,
+            // which we don't want to show to an end user.
+            // Build up the error message manually without the prepended text.
+            return applyError.graphQLErrors
+                .map(({ message }) => message)
+                .join(', ');
+        }
+
+        // A non-GraphQL error occurred.
+        return applyError.message;
+    }, [applyError]);
```

Comment:
I'm pretty certain the identity of `applyError` is changing on every render, so this memoization is not accomplishing much; consider removing.

Reply:
Done in b87b2e757